draft-irtf-cfrg-opaque.md

```diff
@@ -1665,6 +1663,64 @@ learn the client's password. This means that additional checks should be conside
 a production deployment of OPAQUE: for instance, ensuring that there is a
 server-authenticated channel over which OPAQUE registration and login is run.
 
+# API Considerations
+
+## Errors {#api-errors}
+
+Some functions included in this specification are fallible. For example,
+MACs exchanged during the AKE might fail verification, yielding an error. Additionally, protocol messages 
+sent between client and server might not match their expected size, yielding a different error.
+The following errors are documented for the OPRF-3DH composition in this specification and the recommended
+configurations, and do not cover potential errors in other configurations. Participants MUST abort the
+protocol when these errors occur.
+
+### Deserialization errors {#deserialization-errors}
+
+| Name                                 | Role   | Messages                  | Reason                                               |
+|:-------------------------------------|:-------|:--------------------------|:-----------------------------------------------------|
+| InvalidMessageLengthError            | Both   | All                       | The message length is invalid for the configuration. |
+| InvalidBlindedDataError              | Server | RegistrationRequest, Ke1  | Blinded data is an invalid point.                    |
+| InvalidEvaluatedDataError            | Client | RegistrationResponse, Ke2 | Invalid OPRF evaluation point.                       |
```

Comment:
```suggestion
| InvalidEvaluatedDataError            | Client | RegistrationResponse, Ke2 | Invalid OPRF key derivation.                       |
```
Also, it doesn't look like this is actually raised in the spec?

Reply:
Yes, many are implicit.

What about adding a dedicated table for implicit errors coming from other specs?

Reply:
Can't we just add these alongside the existing errors? I think it'd also be good to note these errors alongside errors that OPAQUE generates on its own.